CNV-66741: Add note about single VNC connection (#103044)
openshift/openshift-docs: modules/virt-connecting-vm-virtctl.adoc

+
[source,terminal]
----
$ virtctl vnc <vm_name> -n <namespace> --preserve-session
----
+
where:
+
<vm_name>:: Specifies the name of the VM.
+
<namespace>:: Specifies the namespace that contains the VM.
+

> --preserve-session:: Specifies that if there is an existing connection to the VNC console, it must not be disconnected to allow this session to start.

Reviewer: ```suggestion
--preserve-session:: Prevents an existing VNC console connection from being disconnected when starting a new session.
```

Reviewer: suggestion to tighten the phrasing to active voice
